app/pages/task-list/taskListManifest.json

```diff
@@ -81,6 +81,38 @@
           ]
         }
       ]
+    },
+    {
+      "name": "Add Associated Services",
+      "sections": [
+        {
+          "id": "associated-services",
+          "title": "Add Associated Services to your order",
+          "dependencies": [
+            {
+              "statusDependencies": ["description", "ordering-party", "supplier", "commencement-date", "service-recipients"],
+              "zeroCountDependencies": ["service-recipients"]
+            },
+            {
+              "statusDependencies": ["description", "ordering-party", "supplier", "commencement-date", "service-recipients","catalogue-solutions"],
+              "zeroCountDependencies": ["catalogue-solutions"],
+              "countDependencies": ["service-recipients"]
+            },
+            { 
+              "statusDependencies": ["description", "ordering-party", "supplier", "commencement-date", "service-recipients","catalogue-solutions","additional-services"],
+              "zeroCountDependencies": ["additional-services"],
+              "countDependencies": ["service-recipients","catalogue-solutions"]
+            },
+            { 
+              "statusDependencies": ["description", "ordering-party", "supplier", "commencement-date", "service-recipients","catalogue-solutions","additional-services"],
+              "countDependencies": ["service-recipients","catalogue-solutions","additional-services"]
+            },
+            { 
+              "statusDependencies": ["associated-services"]
+            }
```

Comment:
don't think we need this one

Reply:
We were trying to cover Scenario 8

Scenario 8 - Select no Service Recipients and then go back and add service recipients
Given that a User indicates there are no Service Recipients in the order
and the Associated Service section is enabled
and the User adds an Associated Service
When the User goes back to the Service Recipient section
and adds a Service Recipient
Then the Associated Service section will remain enabled

Reply:
What way would you recommend we ensure that is covered?

Reply:
🤕

Reply:
ah man that scenario is a shambles..  but I think it'll be something like this

```
{ 
              "statusDependencies": ["description", "ordering-party", "supplier", "commencement-date", "service-recipients","associated-services"],
              "countDependencies": ["service-recipients", "associated-services"]
            },
```

Reply:
I need to investigate your recommended change, as it causes a tests to fail which passes with the other version. Will look into it.

Reply:
ok sorted that out